pkg/controller/endpoint/endpoints_controller.go

```diff
@@ -103,9 +104,9 @@ func NewEndpointController(ctx context.Context, podInformer coreinformers.PodInf
 	e.servicesSynced = serviceInformer.Informer().HasSynced
 
 	podInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
-		AddFunc:    e.addPod,
-		UpdateFunc: e.updatePod,
-		DeleteFunc: e.deletePod,
+		AddFunc:    func(obj interface{}) { e.onPodUpdate(nil, obj) },
+		UpdateFunc: e.onPodUpdate,
+		DeleteFunc: func(obj interface{}) { e.onPodUpdate(nil, obj) },
```

Comment:
Should this be `e.onPodUpdate(obj, nil)` when the function has signature of `onPodUpdate(old, cur interface{})`?

Reply:
addressed already in the new rev?